app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<% stylesheet_link_tag "login"%>
+
+<div class="container aligncontent-center" style="width: 500px; flex-direction:cloumn;" >
+  <h2 style='padding:5% 10%'>Log in</h2>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name), :class=> 'container aligncontent-centerr') do |f| %>
+    <div class="form-control-lg">
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: 'Username' %>
+    </div>
+
+    <div class="form-control-lg">
+      <%= f.password_field :password, autocomplete: "current-password", placeholder: 'Password', :class=>' class="form-control-lg"' %>
+    </div>
+
+    <% if devise_mapping.rememberable? %>
+      <div class="form-control-lg rememberable">
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
+      </div>
+    <% end %>
+
+    <div class="form-control-lg align-content-center">
+      <%= f.submit "Log in", :class=>'btn btn-success lg' %>
+    </div>
+  <% end %>
+
+</div>
```

Comment:
- [ ] [REQUIRED] you have actually done a great job implementing devise and I was able to signup and sign in successfully, however, a part of the requirement states that your design for the registration and login have to be modified to match the wireframes and their styling. Let's fix this so it looks like the wireframe.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="1085" alt="Screenshot 2023-07-27 at 13 19 17" src="https://github.com/akos29/blog-app/assets/58692590/c1231c76-85bc-4394-ba40-b0a201ffae2c"> | <img width="1031" alt="Screenshot 2023-07-27 at 13 26 52" src="https://github.com/akos29/blog-app/assets/58692590/348837ea-c15a-44ac-a4d7-d2265e7907f3">